leaflet/static/leaflet/leaflet.extras.js

```diff
@@ -123,10 +128,17 @@ L.Map.DjangoMap = L.Map.extend({
         this.layerscontrol = L.control.layers().addTo(this);
         for (var i = 0, n = layers.length; i < n; i++) {
             var layer = l2d(layers[i]),
+                l = undefined;
+            // Google maps Plugin
+            if (layer.url.substr(0,6) == 'GOOGLE') {
+                l = new L.Google(layer.url.substr(7), layer.options)
```

Comment:
I would prefer not having this duplicated in the code. 

This would also deserve some documentation (like a new section in Readme)